docs/changelogs/v0.23.md

```diff
@@ -83,6 +84,19 @@ attempting to serve data to a browser without a CA certificate.
 
 See [HTTP Gateway over Libp2p](https://github.com/ipfs/kubo/blob/master/docs/experimental-features.md#http-gateway-over-libp2p) for details about this experiment.
 
+#### Removal of `/quic` (Draft 29) support
+
+Kubo no longer supports QUIC Draft 29. This means that older nodes aren't able to connect
+to newer nodes using QUIC Draft 29. However, they are still able to connect through any other
+transport that both nodes talk. QUIC Draft 29 was a preliminary implementation of QUIC before
```

Comment:
What do you think about adding some examples? So people don't have to go so far to see that QUIC is still supported and that just the old version is dropped? 

```suggestion
transport that both nodes talk (such as QUIC RFC 9000, or TCP). QUIC Draft 29 was a preliminary implementation of QUIC before
```

Reply:
Good idea